makefile

```diff
@@ -0,0 +1,20 @@
+SRC = server
```

Comment:
maybe you'd like to share? (-:

https://github.com/Financial-Times/n-makefile/